src/main/features/response/form/models/defendantPaymentOption.ts

```diff
@@ -25,6 +30,25 @@ export class DefendantPaymentType {
       .filter(type => type.value === value)
       .pop()
   }
+
+  displayValueFor (responseType: ResponseType): string {
```

Comment:
is this better than what we had  before?

Reply:
Already discussed but I'll respond for clarity -- it's needed because display value is dynamic depending on response type.